TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,28 @@
+{namespace m=TYPO3\Media\ViewHelpers}
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+  <div class="neos-row-fluid">
+    <div class="neos-span6">
+      <legend>{asset.resource.filename}</legend>
+
+      <table class="neos-info-table">
+        <tbody>
+          <f:for each="{relatedDocumentNodes}" as="node">
+            <tr>
+              <td><neos:link.node node="{node}" /></td>
+              <td class="neos-aRight">{node.nodeData.lastModificationDateTime -> f:format.date(format: 'Y-m-d H:i')}</td>
```

Comment:
If we speak about accessibility, we need a thead on top of the table anyway. Let's do that tomorrow last day of the sprint and @loop86 is not allowed to leave Dresden before removing the WIP tag ;)